test/run.test.ts

```diff
@@ -0,0 +1,123 @@
+import { edkt } from '../src';
+import { AudienceDefinition, PageView } from '../types';
+import { timeStampInSecs } from 'src/utils';
+import { viewStore, audienceStore } from 'src/store';
+
+const sportPageFeatureGetter = {
+  name: 'keywords',
+  func: (): Promise<string[]> => {
+    return Promise.resolve(['sport']);
+  },
+};
+
+const travelPageFeatureGetter = {
+  name: 'keywords',
+  func: (): Promise<string[]> => {
+    return Promise.resolve(['sport']);
+  },
+};
+
+const TTL = 10;
+
+const sportAudience: AudienceDefinition = {
+  id: 'sport_id',
+  name: 'Sport Audience',
+  ttl: TTL,
+  lookback: 10,
+  occurrences: 2,
+  keywords: ['sport'],
+};
```

Comment:
Since this is our e2e test, why do we not use the actual definitions?

The issue here is that the definitions could change, but the test will pass.

Reply:
because I don't want to test our audiences in this test. I have set up the base of the audience test in audience.test.ts

I would still want this test to pass if we decide to change the out the box audiences. This helps keep the test self contained and is checking the run works as expected

Reply:
`audience.test.ts` also does not pull in the actual audience definitions?